Umbraco-Cloud/Set-Up/Media/index.md

```diff
@@ -48,23 +65,23 @@ If you are using IISExpress (as with Visual Studio or WebMatrix) you’ll need t
           </system.webServer>
         </location>
       </configuration>
-      
+
 #### Image Processor
 You’ll need to install the following ImageProcessor packages (latest versions recommended):
 
-* ImageProcessor.Web.4.4.0.0 or later
-* ImageProcessor.Web.Config.2.2.3.0 or later
-* ImageProcessor.Web.Plugins.AzureBlobCache.1.0.4.0 or later
+* ImageProcessor.Web.4.9.3.25 or later
+* ImageProcessor.Web.Config.2.4.1.19 or later
+* ImageProcessor.Web.Plugins.AzureBlobCache.1.4.2.19 or later
 
-These packages are only available via NuGet, so ideally you’ll have your site setup to use Visual Studio.  You can find more information about ImageProcessor and related packages here [https://imageprocessor.org/](https://imageprocessor.org/)
+You can find more information about ImageProcessor and related packages here [https://imageprocessor.org/](https://imageprocessor.org/)
 
-Since Umbraco includes a version of ImageProcessor.Web by default, you will need to update that package first using NuGet:
+Since Umbraco includes a version of ImageProcessor.Web and ImageProcessor.Web.Config by default, you will need to update those packages first using NuGet:
 
 ```PM> Update-Package ImageProcessor.Web```
 
-Then install the other ImageProcessor packages:
+```PM> Update-Package ImageProcessor.Web.Config```
```

Comment:
I just created a brand new project on Cloud, and following this setup it did not have the IP.Web.Config package already, I think the original was right, if you had this installed already it could be from some other package maybe?

We could update this to say install / upgrade the package..